setup.py

```diff
@@ -1,7 +1,5 @@
-import distutils.command
-import distutils.command.build_ext
-import distutils.core
-import distutils.errors
+from setuptools import setup
+from setuptools.extension import Extension
```

Comment:
I agree that the setuptools are not part of standard library yet.

* Because of the [article](http://stackoverflow.com/questions/6344076/differences-between-distribute-distutils-setuptools-and-distutils2). I think it's good to use setuptools. Though, I will add a falling back in next commit.

* There are some setuptools.core.setup() options like install_requires which do not work with distutils.core.setup(). I think it is better to have dependencies to be installed.